book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] As requested by the previous reviewer, Kindly create a method for adding rentals. It may look something like this 👇 

```ruby
def add_rental(person, date)
    Rental.new(date, person, self)
  end